explain/explain.py

```diff
@@ -246,6 +268,36 @@ def cpp_exception_state_present():
         return True
 
 
+def gtest_libraries_present() -> bool:
+    """
+    Is this program linked against gtest libraries?
+    """
+    obj_paths = [Path(o.filename) for o in gdb.objfiles() if o.filename is not None]
```

Comment:
This could be a generator expression to avoid going through all the object files.